src/app/dataset-view/datasetSubs.interface.ts

```diff
@@ -28,3 +28,9 @@ export interface DatasetHistoryUpdate {
     history: MetadataBlockFragment[];
     pageInfo: PageBasedInfo;
 }
+
+export type DataRow = Record<string, unknown>;
```

Comment:
Unused, but I like this name. Maybe `DataRow` should be the name instead of `ObjectInterface`